sherpa/bin/streaming_conformer_rnnt/streaming_server.py

```diff
@@ -0,0 +1,632 @@
+#!/usr/bin/env python3
```

Comment:
Please add documentation to `docs` to show how it can be used.
It can be done in a separate PR.